js/client/modules/@arangodb/testsuites/authentication.js

```diff
@@ -246,8 +228,5 @@ exports.setup = function (testFns, opts, fnDocs, optionsDoc, allTestPaths) {
   testFns['authentication'] = authenticationClient;
   testFns['authentication_parameters'] = authenticationParameters;
 
-  opts['skipAuthentication'] = false;
-
-  for (var attrname in functionsDocumentation) { fnDocs[attrname] = functionsDocumentation[attrname]; }
-  for (var i = 0; i < optionsDocumentation.length; i++) { optionsDoc.push(optionsDocumentation[i]); }
+  tu.CopyIntoObject(fnDocs, functionsDocumentation);
```

Comment:
the call to `CopyIntoList` is missing...

Reply:
this testsuite doesn't have own optionsDocumentations.